ndex2/client.py

```diff
@@ -1077,6 +1593,27 @@ def create_networkset(self, name, description):
         """
         Creates a new network set
 
+	.. code-block:: python
+
+		import ndex2.client as nc
+		import ndex2
+		import networkx as nx
+		import pandas as pd
+		import os
+
+		#create an empty network set in my account set_name is unique. The following code can only run once. 
+		#You have to change the set name if want to retry the code
+		set_name = "MyNetworks"
+		set_description = "a network set including all networks under my username"
+
+		#uri_myset is the URI of the set, 
+		#the format is like http://public.ndexbio.org/v2/networkset/e1310fdf-ba66-11e9-8bb4-0ac135e8bacf
+		uri_myset = my_ndex.create_networkset(set_name, set_description)
```

Comment:
Please add line showing how `my_ndex` is created. The only import needed is the one to import the Ndex2 client